### ISSUE TITLE
Version Problem

### ISSUE THREAD

**Issue by tony37** [cristian99garcia/teachteacher-activity#2]
Version in repository is 3 after several commits. Version 3 ASLO is precedes the sugar3 port.

**Comment by rhl-bthr** [comment]
Thanks, this is a common issue across almost all activities. So I'm closing all such open issues.

My suggestion is to have an automatic sync with GitHub and ASLO;
When a new release tag is set on GitHub, 
 - the activity should be update on ASLO with the latest changes.
 - Issues opened in the repository can go the into known bugs section
 - ReadMe can go into about the Activity section
 - git history or release tag description can be used to explain the new changes

New changes to the repository will have to be tested before setting the new tag

@quozl, please review

**Comment by quozl** [comment]
The exact tooling is irrelevant.  The point that Tony makes is valid; there hasn't been a release, in turn because there hasn't been a maintainer willing to test and tag.  How else do you propose we remember as an organisation that a release is needed?

**Comment by rhl-bthr** [comment]
Since we don't have active maintainers for most of the activities, this problem is bound to happen. An alternate is that reviewers test before merging each change, and release when they deem fit. Automation will reduce their work

**Comment by quozl** [comment]
@pro-panda, how else do you propose we remember as an organisation that a release is needed?  The mass deletion of Tony's careful checking seems wrong to me.

**Comment by rhl-bthr** [comment]
In retrospect, I should have waited for consensus before closing all issues. I'll reopen all issues.

An interesting thought,
 - Sugar or ASLO can allow users to opt for the beta version updates for activities, which can be in sync with every commit on GitHub
 - Normal users get updates only on new tags

**Comment by quozl** [comment]
> the beta version

Yes, interesting.  However this would make merging more likely to give unintended consequences.  I would not want to hinder every patch by requiring testing.  For certain activities, if we switched to test everything (like CI), instead of integration testing, there would be more noise and work created.
